src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -1,27 +1,92 @@
 import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
 
+import Artwork from "../../../../components/artwork"
 import Button from "../../../../components/buttons/inverted"
 import Nav from "../../../../components/nav"
 import NavItem from "../../../../components/nav_item"
 import TextArea from "../../../../components/text_area"
 import Title from "../../../../components/title"
 
+const InquiryContainer = styled.div`
+  display: inline-block;
+`
+
+const Container = styled.div`
+  text-align: center;
+
+  & .footer {
+    max-width: 500;
+    margin: 10px auto;
+  }
+`
+
 class Inquiries extends React.Component<any, any> {
+  renderArtworks() {
+    if (!this.props.user) {
+      return []
+    }
+
+    const edges = this.props.user.artwork_inquiries_connection.edges || []
+    return edges.map(edge => {
+      const artwork = edge.node.artwork
+      return (
+        <InquiryContainer>
+          <Artwork key={artwork.id} artwork={artwork} />
```

Comment:
Btw,`artwork.id` needs to be a global id across the app. Should use `__id` from relay or another global id.